platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_ssd_upgrade_status.sh

```diff
@@ -50,13 +50,13 @@ if [ $SSD_UPGRADE_STATUS1 == "2" ]; then
 
     echo "$0 `date` Upgraded to unknown version after first mp_64 upgrade." >> $SSD_UPGRADE_LOG
 
-elif [ $SSD_UPGRADE_STATUS2 == "2" ];then
+elif [ $SSD_MODEL == "3IE3" ] && [ $SSD_UPGRADE_STATUS2 == "2" ];then
     rm -rf $SSD_FW_UPGRADE/GPIO7_*
     touch $SSD_FW_UPGRADE/GPIO7_error
 
     echo "$0 `date` Upgraded to unknown version after second mp_64 upgrade." >> $SSD_UPGRADE_LOG
```

Comment:
Can we combine these first two error cases since they are performing exactly same process and add some more information? And what unknown version could be upgraded for this fw-upgrade case? Probably indicating the device has already some unknown version installed?

Reply:
@sujinmkang These conditions are listed separately to have the debug information on firmware upgrade in case a failure is seen which would not be hit otherwise.